pulser/sequence.py

```diff
@@ -256,6 +275,36 @@ def current_phase_ref(self, qubit: QubitId,
 
         return self._phase_ref[basis][qubit].last_phase
 
+    def set_magnetic_field(self, bx: float = 0., by: float = 0.,
```

Comment:
I think now that since this is given in Gauss, perhaps 1 is too little, this is my bad. Published work shows values from 10 to 50 G for the magnetic field. I suggest we set 30 G for the moment.